README.md

```diff
@@ -11,10 +11,28 @@ $ npm install openwhisk
 
 ## usage
 
+Instantiate openwhisk with specific options:
+
 ```
-var openwhisk = require('openwhisk');
-var options = {apihost: 'openwhisk.ng.bluemix.net', api_key: '...'};
-var ow = openwhisk(options);
+function main(params) {
+	// require the openwhisk npm package
+	var openwhisk = require('openwhisk');
+
+	// read apihost, auth, and namespace from params
+	var apiHost = params.apiHost; # e.g. openwhisk.ng.bluemix.net
+	var namespace = params.namespace;
+	var auth = params.auth; 
+
+	// generate api_key from auth
+	var base64Auth = new Buffer(auth).toString('base64');
+	var apiKey = "Basic " + base64Auth;
+
+	var options = {apihost: apiHost, api_key: apiKey, namespace: namespace};
+	
+	// instantiate the openwhisk instance before you can use it
+	var wsk = openwhisk(options);
+	...
+}
 ```
```

Comment:
In this OpenWhisk environment, the client library will automatically pull the credential values from environment values. The developer does not need to pass in any parameters. I should add an example to make this clear.